### DISCUSSION TITLE
Can't view "Find all references"

### DISCUSSION THREAD

**Discussion by sldev2021** [alefragnani/vscode-language-pascal#132] [Q&A] [answered]
Hi, Alessandro,
thanks again for your work,
I think to have installed everything correctly(no error or message when open a folder) code navitagion ok...succesfully go to definition etc...
but I can't view "Find all references" command, what to do for enable this function ?

this is my menu on mouse right click

![immagine](https://user-images.githubusercontent.com/129405429/231525670-ed116f34-5cae-4653-9c92-5c9dac9353ac.png)

Thanks in advance,
Regards

**Comment by alefragnani** [comment, marked as answer]
Hi @sldev2021 ,

There is nothing special to make `Find All References` to be show, once you leave `pascal.codeNavigation` as `workspace`. It should just works.

But I wonder if you are also using _another_ extension as well, because the `Go to Implementations` and `Find All Implementations` are not from my Pascal extension. Is VS Code displaying `Pascal` or `ObjectPascal` in the status bar, for pascal related files? If it displays `ObjectPascal`, it is not handled by this extension.

Hope this helps
